### ISSUE TITLE
Error writing .mh_sequences file

### ISSUE THREAD

**Issue by scanner** [scanner/asimap#392]
Somewhere bad data got in to the .mh_sequences file. Probably somewhere we are not converting `\Seen` to `Seen`:
```
Traceback (most recent call last):
  File "/app/asimap/mh.py", line 376, in aget_sequences
    name, contents = line.split(":")
    ^^^^^^^^^^^^^^
ValueError: not enough values to unpack (expected 2, got 1)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/app/asimap/user_server.py", line 274, in run
    await self.cmd_processor.command(imap_cmd)
  File "/app/asimap/client.py", line 185, in command
    result = await getattr(self, f"do_{imap_command.command}")(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/asimap/client.py", line 1196, in do_fetch
    async for idx, results in self.mbox.fetch(
  File "/app/asimap/mbox.py", line 2114, in fetch
    iter_results.append(await elt.fetch(ctx))
                        ^^^^^^^^^^^^^^^^^^^^
  File "/app/asimap/fetch.py", line 187, in fetch
    msg = await self.ctx.email_message()
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/asimap/search.py", line 157, in email_message
    message_from_string(msg_as_string(await self.msg()), policy=SMTP),
                                      ^^^^^^^^^^^^^^^^
  File "/app/asimap/search.py", line 138, in msg
    self._msg = await self.mailbox.mailbox.aget_message(self.msg_key)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/asimap/mh.py", line 254, in aget_message
    sequences = await self.aget_sequences()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/asimap/mh.py", line 391, in aget_sequences
    raise FormatError(
mailbox.FormatError: Invalid sequence specification: 3472-3476
```
File contents:
```
$NotJunk: 1-723 725-913 921-1128 1130-1343 1345 1347 1351-1962 1965-2042 2044-2050 2052-2264 2266 2268-2312 2315-2526 2528-26
17 2621-2629 2631-2635 2638 2640-2641 2645-3081 3083-3108 3110-3112 3114 3116-3117 3119 3121-3123 3125 3128-3130 3132-3133 31
35-3136 3139-3158 3160 3162-3166 3172-3173 3175-3187 3189-3192 3197-3225 3227-3245 3248-3313 3315-3394 3396-3442 3444-3445 34
47-3464 3466-3471 3474
Seen: 1-1128 1130-1349 1351-2526 2528-3158 3160 3162-3464 3466-3476
NotJunk: 1-723 725-913 921-1128 1130-1343 1345 1347 1351-1962 1965-2042 2044-2050 2052-2264 2266 2268-2312 2315-2526 2528-261
7 2621-2629 2631-2635 2638 2640-2641 2645-3081 3083-3108 3110-3112 3114 3116-3117 3119 3121-3123 3125 3128-3130 3132-3133 313
5-3136 3139-3158 3160 3162-3166 3172-3173 3175-3187 3189-3192 3197-3225 3227-3245 3248-3313 3315-3394 3396-3442 3444-3445 344
7-3464 3466-3471 3474
Junk: 956 1719
$Junk: 956 1719
replied: 2468
$Forwarded: 2513 2524-2525 2574
Forwarded: 2513 2524-2525 2574
\Seen: 3309-3464 3466-3471 3474-3475
Recent: 3472-3476
3472-3476
```

**Comment by scanner** [comment]
Either it is related or a bigger problem is the appearence of `\Seen` in the sequences. Need to figure out how that is getting in there.

**Comment by scanner** [comment]
Fixed in #398
